client/src/context/useAuthContext.tsx

```diff
@@ -4,25 +4,29 @@ import { AuthApiData, AuthApiDataSuccess } from '../interface/AuthApiData';
 import { User } from '../interface/User';
 import loginWithCookies from '../helpers/APICalls/loginWithCookies';
 import logoutAPI from '../helpers/APICalls/logout';
+import { ProfileApiDataSuccess } from '../interface/ProfileApiData';
+import { Profile } from '../interface/Profile';
 
 interface IAuthContext {
   profile: any;
   loggedInUser: User | null | undefined;
   updateLoginContext: (data: AuthApiDataSuccess) => void;
+  updateProfileContext: (data: ProfileApiDataSuccess) => void;
   logout: () => void;
 }
 
 export const AuthContext = createContext<IAuthContext>({
   profile: undefined,
   loggedInUser: undefined,
   updateLoginContext: () => null,
+  updateProfileContext: () => null,
   logout: () => null,
 });
 
 export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
   // default undefined before loading, once loaded provide user or null if logged out
   const [loggedInUser, setLoggedInUser] = useState<User | null | undefined>();
-  const [profile, setProfile] = useState();
+  const [profile, setProfile] = useState<Profile | null | undefined>();
```

Comment:
```suggestion
  const [loggedInUserProfile, setLoggedInUserProfile] = useState<Profile | null | undefined>();
```
Good thinking adding the Profile to this context, let's change the names of the state to avoid any confusion 👍🏽